### PR TITLE
kubectl_kadalu python2 support

### PR DESCRIPTION
**Pull request by vatsa287**
Signed-off-by: vatsa287 <i.mnshreevatsa@gmail.com>

### CODE REVIEW

File: cli/kubectl_kadalu/install.py [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
 from kubectl_kadalu import utils
 
-
 def install_args(subparsers):
     """ add arguments to argparser """
     parser_install = subparsers.add_parser('install')
     parser_install.add_argument(
         "--version",
         help="Kadalu Version to Install [default: latest]",
-        choices=["0.6.0", "master", "latest"],
```
Comment (amarts) [inline cli/kubectl_kadalu/install.py]: please keep 'master' there for now!

File: cli/kubectl_kadalu/utils.py [hunk 2]
```diff
@@ -3,23 +3,29 @@
...
         self.stderr = err
 
 
 class CommandError(Exception):
     """ Class for handling exceptions """
-    def __init__(self, rc, err):
-        super().__init__("error %d %s" % (rc, err))
-        self.returncode = rc
+    def __init__(self, returncode, err):
+        # if sys.version_info >= (3,):
```
Comment (aravindavk) [inline cli/kubectl_kadalu/utils.py]: also remove these commented lines

Comment (vatsa287) [reply cli/kubectl_kadalu/utils.py]: ok sure

File: cli/kubectl_kadalu/install.py (hunk 1 above)
Comment (vatsa287) [reply cli/kubectl_kadalu/install.py]: okay

**Review by amarts — APPROVED** [review]

**Comment by amarts** [comment]
cli/kubectl_kadalu/utils.py:9:0: R0205: Class 'CmdResponse' inherits from object, can be safely removed from bases in python3 (useless-object-inheritance)
Makefile:41: recipe for target 'pylint' failed

**Review by amarts — COMMENTED** [review]
see the pylint error!

**Review by amarts — APPROVED** [review]

**Pull request merged**